src/System.Runtime.Extensions/tests/System/Convert.ToBase64CharArray.cs

```diff
@@ -94,5 +94,18 @@ public static void InvalidInputLength()
             Assert.Throws<ArgumentOutOfRangeException>(() => Convert.ToBase64CharArray(inputBytes, 0, inputBytes.Length + 1, outputBuffer, 0));
             Assert.Throws<ArgumentOutOfRangeException>(() => Convert.ToBase64CharArray(inputBytes, 1, inputBytes.Length, outputBuffer, 0));
         }
+
+#if netstandard17
```

Comment:
I would put this test in with the Dbunull tests and call it Convert.netstandard1.7